### ISSUE TITLE
Parts list/Instructions 

### ISSUE THREAD

**Issue by americoperez49** [misprit7/real-time-chess#2]
Hey From Scratch,

Just came across your video for this. Looks super fun!

Would you be able to provide a complete parts list with links to where we could buy/manufacture the parts from?

Also a tutorial for how to build/assemble this would also be super helpful.

Thank you!

**Comment by androng** [comment]
How much does one build cost?

**Comment by Jannis-L** [comment]
> Hey From Scratch,
> 
> Just came across your video for this. Looks super fun!
> 
> Would you be able to provide a complete parts list with links to where we could buy/manufacture the parts from?
> 
> Also a tutorial for how to build/assemble this would also be super helpful.
> 
> Thank you!

A bill of materials can be found in the "releases", however the used electric magnets are not listed there. A google search resulted in all sorts of different versions that would fit the PCB design.
Additionally, there are no CAD-Files for the milled plates.

However, it seems like the project isn't ready for being copied, as the PCB is very much in a prototype state:
- screws that can short traces
- unoptimzed layout
- edges could be smaller
- not splitting the PCB in 4 parts would make it cheaper (only one controller needed) and easier to assemble. (I understand that it made sense for the first prototype)
- Having a small display with two buttons would allow for a ton of different online-only chess-variants to be played on it.
- The all-metal construction makes the project to expensive and hard to rebuild.
- In the video the board seems to be unreliable at detecting a piece being placed on a field and/or pulling it in place correctly.

The question on my mind: Is @misprit7 willing to maintain and update the project (or at least manage pull requests) to get it to a state where everyone can build their own version for a few hundred dollars or should we do a public fork and try to develop it further ourselves?

**Comment by Sooly890** [comment]
Yes... please a guide would be useful

**Comment by misprit7** [comment]
I added a link to the onshape mechanical files and the other parts I used in the readme. I won't be providing a guide, although I think most parts are reasonably self explanatory. If anyone does actually try to build one and runs into any issues with the files let me know any specific questions and I'd be happy to help.

As for cost, I'd probably expect at least $2500 for the first board. ~$700 for the electromagnets, ~$1000 for the cnced parts, ~$100 for the pieces, ~$300 for pcbs+components, plus a couple hundred for fasteners/materials/misc.

**Comment by danaedf** [comment]
Interesting project! 

If you make a new board revision, I would recommend increasing the layer count to 4 for dedicated GND/PWR layers. This should help with the voltage drops you have noticed when multiple pieces are on cooldown. Of course, you may decide against this, depending on the price, but as you mentionned in your readme, the 5V traces would need to be much larger if you keep it at 2 layers.

Also, decoupling capacitors near the electromagnets might be nice, because the in-rush current could drop the bus voltage enough to reset nearby LEDs. On the other hand, a high enough LED refresh rate might make this potential flicker unnoticeable. Adding footprints (even if you decide to not install them) for ceramic capacitors of 0.1uF for each LED and a 10uF near the eletromagnet's 5V would give you options if you notice issues.

**Comment by misprit7** [comment]
@danaedf Yes you're totally right gnd/pwr planes are better for this, I remember regretting not putting them in but forgot to mention it when I wrote the last message. As for capacitors I haven't had any issues, although definitely something I was a bit worried about and would be reasonable to add.

**Comment by Tomillium** [comment]
hey, could you provide a file with the pcb design.
-thanks

**Comment by Tomillium** [comment]
Hey, I was wondering if the LEDs came on the pcb or not. 
also if they don't which ones did you use?
